src/ignitus-Shared/Components/InternshipList/index.ts

```diff
@@ -0,0 +1,43 @@
+import styled from '@emotion/styled';
+import * as T from '../emotionStyles/shared'
+import * as C from '../emotionStyles/colors'
+
+export const InternshipTitle = styled(T.Title)`
+  color:orange
+`;
+
+export const SubHeading = styled(T.Title)`
+  font-size: 1.2em;
+`
+
+export const Description = styled(T.Paragraph)`
+	color: grey;
+	margin-bottom: 3px;
+`
+
+export const DateInfo = styled(T.Paragraph)`
+	color: grey;
+	right: 0;
+	position: absolute;
+	top: 10px;
+	padding: 10px;
+	@media (max-width: 576px) {
```

Comment:
use something like this for defining `mediaQueries` https://github.com/Ignitus/Ignitus-client/blob/master/src/ignitus-Shared/Components/emotionStyles/shared.ts#L5